internal/sql/repository/pipelineConfig/CiPipelineMaterial.go

```diff
@@ -112,17 +113,17 @@ func (impl CiPipelineMaterialRepositoryImpl) Save(tx *pg.Tx, material ...*CiPipe
 	return err
 }
 
-func (impl CiPipelineMaterialRepositoryImpl) Update(tx *pg.Tx, materials ...*CiPipelineMaterial) error {
-	/*err := tx.RunInTransaction(func(tx *pg.Tx) error {
-		for _, material := range materials {
-			r, err := tx.Model(material).WherePK().UpdateNotNull()
-			if err != nil {
-				return err
-			}
-			impl.logger.Infof("total rows saved %d", r.RowsAffected())
+func (impl CiPipelineMaterialRepositoryImpl) UpdateForSwitch(tx *pg.Tx, materials ...*CiPipelineMaterial) error {
+	for _, material := range materials {
+		err := tx.Update(material)
```

Comment:
go-pg supports bulk update, use that instead of iterating through loop

Reply:
with this change we should also rename the method to something like UpdateInBulk, BulkUpdate etc. to make this method generic and less confusing for others to use